.github/workflows/release-python.yml

```diff
@@ -0,0 +1,115 @@
+name: Release Python
+
+on:
+  push:
+    tags:
+      - python/v*
+
+jobs:
+  macos:
+    runs-on: macos-11
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: aarch64-apple-darwin
+          profile: minimal
+          default: true
+      - name: Build
+        run: cargo build --release
```

Comment:
I don't think this is necessary for building wheels. Is there a specific reason this is here?

Reply:
At first I thought it might make debugging easier by having discrete steps for building the Rust code and the wheels but it wasn't particularly helpful. I'll remove it.